Protobuild.toml

```diff
@@ -19,7 +19,7 @@ plugins = ["grpc"]
   # This is the default.
   after = ["/usr/local/include"]
 
-# This section let's us map protobuf imports to Go packages. These will become
+# This section let's map protobuf imports to Go packages. These will become
```

Comment:
The original statement is in the present tense, so `s/let's us/lets us/`. Could also be rephrased as `allows us to`.